src/integration/spring/src/jvmMain/kotlin/community/flock/wirespec/integration/spring/kotlin/client/WirespecClient.kt

```diff
@@ -0,0 +1,73 @@
+package community.flock.wirespec.integration.spring.kotlin.client
+
+import community.flock.wirespec.kotlin.Wirespec
+import community.flock.wirespec.kotlin.Wirespec.Serialization
+import kotlinx.coroutines.reactor.awaitSingle
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpMethod
+import org.springframework.http.HttpStatusCode
+import org.springframework.web.reactive.function.client.WebClient
+import reactor.core.publisher.Mono
+
+class WirespecClient(
```

Comment:
Maybe WirespecWebClient is a better name

Reply:
Good suggestion, since client is already a concept in Wirespec